libraries/AP_HAL_ChibiOS/hwdef/MambaF405/hwdef.dat

```diff
@@ -0,0 +1,168 @@
+# hw definition file for processing by chibios_pins.py
+# Mamba F405 MK2
+# with F405 mcu, mpu6000 imu and 7456 series osd
+@ thanks to betaflight for pin information
+
+# MCU class and specific type.
+MCU STM32F4xx STM32F405xx
+
+HAL_CHIBIOS_ARCH_F405 1
+
+# board ID for firmware load
+APJ_BOARD_ID 1019
+
+# crystal frequency
+OSCILLATOR_HZ 8000000
+STM32_PLLM_VALUE 8
+
+# board voltage
+STM32_VDD 330U
+
+STM32_ST_USE_TIMER 5
+
+# order of I2C buses
+I2C_ORDER I2C1 I2C2
+
+# order of UARTs (and USB)
+SERIAL_ORDER OTG1 USART1 EMPTY USART3 EMPTY EMPTY USART6
```

Comment:
why is USART6 right at the end? why not in as SERIAL2 ?

Reply:
As discussed its sufficient to simply document the current order in the hwdef.dat

Reply:
Perhaps add a comment that this is the order in the documentation